### DISCUSSION TITLE
Works great except for Samsung Internet browser and social app browsers

### DISCUSSION THREAD

**Discussion by RobF3000** [thumbmarkjs/thumbmarkjs#54] [General]
Hi all,

This is a great alternative to Fingerprint! 

The only issues that we have found is that it doesn't work on Samsung Internet mobile browser, as well as when visiting the website from a link in Instagram or X/Twitter mobile apps (and therefore their browsers) - hoping maybe one day it will work with those as well.

It might not work from other social app browsers but haven't tested any others yet.

Thanks, just thought I'd post here for informative purposes - I'm new to GitHub, hoping this post is ok and within guidelines :)

**Comment by ilkkapeltola** [comment]
Absolutely. Thanks for the comment.

Does it not give anything on those browsers?
